app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -178,9 +197,9 @@ fun Imagery() {
       modifier = Modifier.fillMaxWidth().height(200.dp).padding(3.dp),
   ) {
     items(com.android.sample.ui.activity.items.size) { index ->
-      Image(
-          painter = painterResource(id = com.android.sample.ui.activity.items[index].imageResId),
-          contentDescription = com.android.sample.ui.activity.items[index].contentDescription,
+      AsyncImage(
```

Comment:
For the moment, only using URL of images. This not a very good idea because it is not a natural way to deal with images. We should definitively find a more natural way to deal with images.